cli/azd/.vscode/cspell-azd-dictionary.txt

```diff
@@ -132,6 +132,7 @@ swacli
 Syncer
 teamcity
 testdata
+tmpl
```

Comment:
Do we need to update the aks todo templates to support this?

Reply:
That's what #2460 is doing.

Reply:
Yes, see linked PR #2460